avvoenv.1.ronn

```diff
@@ -53,6 +53,12 @@ current service, or canonicalise the name given with the `--service` option.
     Set the Consul URL, overriding the `CONSUL_HTTP_ADDR` environment variable,
     and the default of <http://127.0.0.1:8500>.
 
+  * `--app-id` [<app-id>] `--app-user` [<app-user>]:
```

Comment:
Even though these are related I think they should be documented separately, to match the existing format / general man page format. They should also appear before `--consul` so it's alphabetical. Remember to add them to the synopsis section too.